spotifyutils/__init__.py

```diff
@@ -51,6 +52,31 @@ def cli(input_args=None):
     args = parser.parse_args(input_args)
     main(**vars(args))
 
-
 def main(**kwargs):
-    print('Test')
+    """ Direct user to authorize spotify URL """
+
+    configfile = kwargs['configfile']
+    
+    config = configparser.ConfigParser()
+    config ['spotifyutils'] = {
+        'ClIENT_ID': kwargs['client_id'],
+        'CLIENT_SECRET': kwargs['client_secret'],
+        'REDIRECT_URI': kwargs['redirect_uri']
+    }
+
+    with open(configfile, "w") as f:
```

Comment:
Why is the file being written here? It should only be read unless the user explicitly operates in a configuration mode, or edits the file directly.

Reply:
In the most recent version of code, the "configfile" arg is completely optional. If a user passes a configfile arg along with other args, the args passed are written to the configfile of the users choosing. If the configfile arg is passed with no accompanying args, the configfile is read and the values are pulled. If no configfile arg is passed, only the CLI args are used.